djangosaml2/views.py

```diff
@@ -378,6 +378,14 @@ def logout(request, config_loader_path=None):
 
     result = client.global_logout(subject_id)
 
+    try:
+        result = client.global_logout(subject_id)
+    except Exception as exp:
```

Comment:
Could you catch the more specific exception type, instead of a broad exception handler, or are there too many different possibilities?

Reply:
I didn't dig into possibilities, so I managed It as It Is.

Reply:
Would you mind digging (if it's still relevant) please?
I would prefer to avoid catching all exceptions like this. Thanks.

Reply:
I really don't know when I'll be able to back on this, if possibile I will

Reply:
LogoutError exception catched from pysaml2 client, done.